SpecialManageWiki.php

```diff
@@ -109,6 +109,14 @@ function showWikiForm( $wiki ) {
 				'options' => $options,
 				'name' => 'mwLanguage',
 			),
+			'logo' => array(
+				'label-message' => 'managewiki-label-logo-url',
+				'type' => 'text',
+				'size' => 20,
+				'default' => 'https://static.miraheze.org/metawiki/3/35/Miraheze_Logo.svg',
```

Comment:
The problem with this is we'd want people to only be able to select a logo if it's uploaded under static.miraheze.org, so people can't set external links as logos. I think there should be a way to force this. If not make users write File:Logo and then convert that to the actual link.